app/components/gutenberg/Block/Block.tsx

```diff
@@ -156,7 +156,7 @@ RegisteredBlocks["gutenberg-ant-design/col"] = {
   }: RegisteredBlocksComponentProps & {
     block: GutenbergAntDesignColBlockProps;
   }) => {
-    const Col = dynamic(() => import("antd").then((mod) => mod.Col));
+    const Col = dynamic(() => import("antd").then((mod) => mod.Col), {ssr: false});
```

Comment:
@Danishkhurshid Can you add a comment above this line explaining the reasoning, would be good for reference in the future?